scripts/verify-dunder-init.sh

```diff
@@ -38,7 +38,7 @@ exclude+='|^common/test/data/?.*$'
 #  * common/lib/xmodule -> EXCLUDE from check.
 #  * common/lib/xmodule/xmodule/modulestore -> INCLUDE in check.
 exclude+='|^common/lib$'
-exclude+='|^common/lib/(capa|safe_lxml|sandbox-packages|symmath|xmodule)$'
```

Comment:
Grepping for 'symmath', I found one other instance like this you can clean up in `scripts/post-pip-compile.sh`.

Reply:
The mention, and the associated logic have been removed.